src/services/i18n.service.ts

```diff
@@ -4,11 +4,15 @@ import * as path from 'path';
 import { I18nService as I18nServiceAbstraction } from 'jslib/abstractions/i18n.service';
 
 // First locale is the default (English)
-const SupportedTranslationLocales = [
-    'en', 'cs', 'da', 'de', 'es', 'et', 'fi', 'fr', 'hr', 'hu', 'id', 'it', 'ja',
-    'nb', 'nl', 'pl', 'pt-BR', 'pt-PT', 'ro', 'ru', 'sk', 'sv', 'tr', 'uk', 'vi',
-    'zh-CN', 'zh-TW',
-];
+export const SupportedLocalesLanguage = {
```

Comment:
Can you revert this back to a standard string array? We will need to support localized strings for the language names.